README.md

```diff
@@ -1,19 +1,44 @@
-# App ID module
-<!-- UPDATE BADGE: Update the link for the following badge-->
-[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+<!-- Update the title -->
+# IBM AppID module
+
+<!--
+Update status and "latest release" badges:
+  1. For the status options, see https://terraform-ibm-modules.github.io/documentation/#/badge-status
+  2. Update the "latest release" badge to point to the correct module's repo. Replace "terraform-ibm-module-template" in two places.
+-->
+[![Implemented (No quality checks)](https://img.shields.io/badge/Status-Implemented%20(No%20quality%20checks)-yellowgreen)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
```

Comment:
This PR has quality checks (you have tests). So suggest updating to stable badge